framework/src/service/ServiceReferenceBasePrivate.cpp

```diff
@@ -64,9 +67,9 @@ InterfaceMapConstPtr ServiceReferenceBasePrivate::GetServiceFromFactory(
                                                     ServiceRegistrationBase(registration));
     if (!smap || smap->empty())
     {
-      registration->bundle->coreCtx->listeners.SendFrameworkEvent(FrameworkEvent(FrameworkEvent::Type::FRAMEWORK_WARNING, 
-                                                                                  MakeBundle(bundle->shared_from_this()), 
-                                                                                  std::string("ServiceFactory returned an empty or nullptr interface map."), 
+      registration->bundle->coreCtx->listeners.SendFrameworkEvent(FrameworkEvent(FrameworkEvent::Type::FRAMEWORK_WARNING,
```

Comment:
True, on the other hand, I didn't want to change this because it is not related to the actual task being worked on. I will change the documentation instead and review the event codes later in #217.